sample-apps/react-demo/app/ClientApp.tsx

```
@@ -74,6 +74,16 @@ export const ClientApp = ({ children }: PropsWithChildren) => {
     },
   });
 
+  useEffect(() => {
+    const off = client?.on('errors.unhandled', async (_) => {
+      console.error('errors.unhandled', _);
+      await client?.disconnectUser();
+      await client?.connectUser(CURRENT_USER, CURRENT_USER.token);
+    });
+
+    return () => off?.();
+  }, [client]);
```

ESLint

[error] React Hook useEffect has a missing dependency: 'CURRENT_USER'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)